### PR TITLE
More Updates

### PR DESCRIPTION
**Pull request by varun-s22**
In this PR, I have added a Create Invite button for the room admin, to send invites to others. The button is only visible to room owner. The invite page is overlaid over the chat window page, and has a new route all over. 

The Login/SignUp form has now a view password button (eye icon) helping users to see their typed password. For this I created a new component called PasswordField, taking the InputField and eye images as imports and exporting them layered in a span element.

Some minor fixes, code refactoring, and changed online status colour to light green.

This is how it looks currently
Screenshot 2022-08-03 at 15 46 45

### CODE REVIEW

File: front-end/src/Pages/DialogBox.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { useState } from 'react'
+export interface Dialog {
+  id: string
+  name: string
+}
+export interface IDialogBoxProps {}
+const DialogBox = (props: IDialogBoxProps) => {
+  const params = useParams<Dialog>()
+  const history = useHistory()
+  const link = window.location.protocol + '//' + window.location.host + `/group/${params.id}`
```
Comment (TheTrio) [inline front-end/src/Pages/DialogBox.tsx:13]: This should be its own function

File: front-end/src/components/LeftPane/Users/LeftPane.tsx [hunk 2]
```diff
@@ -41,10 +45,14 @@ export default function LeftPane(props: ILeftPaneProps) {
             </li>
           ))}
       </div>
-      {currentUser !== roomOwner?.username && (
+      {currentUser !== roomOwner?.username ? (
         <div className={styles['exit-group']}>
```
Comment (TheTrio) [inline front-end/src/components/LeftPane/Users/LeftPane.tsx:49]: Use a different class name here. This is no longer for exiting the group

File: front-end/src/components/UI/PasswordField.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
```
Comment (TheTrio) [inline front-end/src/components/UI/PasswordField.tsx:1]: This is redundant

File: front-end/src/components/UI/PasswordField.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import styles from './PasswordField.module.css'
+import { useState } from 'react'
```
Comment (TheTrio) [inline front-end/src/components/UI/PasswordField.tsx:3]: All the React imports should be on the same line

**Review by TheTrio — CHANGES_REQUESTED** [review]

**Comment by varun-s22** [comment]
I've changed the files accordingly, you may now review again

**Pull request merged**